applications/plugins/image/CMakeLists.txt

```diff
@@ -73,6 +73,13 @@ set(README_FILES
     image.txt
     )
 
+
+if(OpenGL_FOUND)
```

Comment:
Maybe a small message to inform the user that OpenGL was not found, thus not activating some features (e.g DepthMapToMeshEngine)?

Reply:
SOFA_NO_OPENGL and (NOT OpenGL_FOUND) are not really the same.
One is a choice, the other one is a dependency.
You can want to compile SOFA without any OpenGL but compile the image plugin with the OpenGL features (weird but hey, can happen I guess?)